src/dev/i18n/extract_default_translations.js

```diff
@@ -38,15 +40,56 @@ function addMessageToMap(targetMap, key, value) {
   targetMap.set(key, value);
 }
 
-export async function extractDefaultTranslations(inputPath) {
+async function getPluginsPaths(inputPaths) {
```

Comment:
__nit:__ it seems you don't need `async` for this function and hence `await` where you call it.